.github/workflows/versions.yaml

```diff
@@ -6,8 +6,18 @@ on:
 jobs:
   versions:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        branch:
+          - 'release-0.5'
+          - 'release-0.6'
+          - 'release-0.7'
+          - 'release-0.8'
```

Comment:
How do we plan of keeping this list up to date?

Can we test this only on one additional branch first?

Reply:
It should be the same process as for the e2e tests that we update based on the compatibility matrix. I don't think the process of updating the compatibility matrix is detailed somewhere, but we might want to add some documentation there since there are a couple of things to update alongside it.

Yeah sure, I will update the PR to only add `release-0.8`.